### PR TITLE
Tabs > Spaces

### PR DESCRIPTION
**Pull request by celicoo**
Richard is right! :P

### CODE REVIEW

**Comment by Stitchpunk** [comment]
PR conflicts with master

**Comment by celicoo** [comment]
@Stitchpunk Conflicts solved ;)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
Comment (aisk) [inline .editorconfig]: The actual indent is 2 in the `common.cc`.

**Comment by Stitchpunk** [comment]
Whoops, let's try one more time

**Comment by celicoo** [comment]
@Stitchpunk Done 🎯

**Comment by celicooo** [comment]
Ihaaaa!!! Thanks @Stitchpunk  <3

**Comment by celicoo** [comment]
Ihaaa!!! Thanks @Stitchpunk  <3

**Pull request merged**